### ISSUE TITLE
Unknown CMake generator "Visual Studio 14 2015 Win64"

### ISSUE THREAD

**Issue by assumptionsoup** [vector-of-bool/vscode-cmake-tools#11]
I'm getting the error `Unknown CMake generator "Visual Studio 14 2015 Win64"` when I add this to my settings.json file and run cmake configure:
```
"cmake.preferredGenerators": [
    "Visual Studio 14 2015 Win64"
]
```
`cmake -G "Visual Studio 14 2015 Win64"` works on the command line (git bash), so I'm not sure what's going on here.  The cmake extension falls back to "Visual Studio 14 2015", which is wrong in this case - I need a 64bit build.

**Comment by vector-of-bool** [comment]
This looks like it will probably be fixed by #10. I'll be able to give it a better look later today. Unfortunately, it's difficult for me to test on Windows as I only run a Linux box at home.

A workaround is to run the initial CMake command from the command line and then open up the project in Code. It should continue to use the generator you requested.

**Comment by assumptionsoup** [comment]
Wow, good timing.  Thanks for looking into it.  I'll use the workaround until #10 is approved/released.
